src/components/app/Root.jsx

```diff
@@ -34,7 +35,7 @@ const Root = () => {
       }
     }, NPROGRESS_DELAY_MS);
     return () => clearTimeout(timeoutId);
-  }, [navigation, fetchers]);
+  }, [navigation, fetchers, authenticatedUser]);
```

Comment:
nit: probably can do without adding `authenticatedUser` to the dependency array if not using it quite yet :)